src/PredictorMfe2dHeuristic.cpp

```diff
@@ -143,13 +143,20 @@ fillHybridE()
 
 		} // w2
 		} // w1
-
+		curCellArr[i1][i2-1].E=curCell->E;
+		curCellArr[i1][i2-1].j1=curCell->j1;
+		curCellArr[i1][i2-1].j2=curCell->j2;
+		curCellEtotalArr[i1][i2-1] = curCellEtotal;
```

Comment:
as stated for line 158, both arrays are obsolete and should be removed (computation and memory overhead)